### PR TITLE
Update RTL guidance documents

### PR DESCRIPTION
**Pull request by simonguo**

### CODE REVIEW

**Comment by simonguo** [comment]
Deploy preview for *rsuite* processing.

Building with commit efd059a8dd8b142d7c011c36ecf72f7b5efb53ff

https://app.netlify.com/sites/rsuite/deploys/5e0ed43fecda3f0009737608

File: src/pages/guide/rtl/en/index.md [hunk 1]
```diff
@@ -12,22 +12,27 @@ Make sure the `dir` attribute is set on the body:
...
-  </IntlProvider>,
-  document.getElementById('root')
-);
+function RTL(props) {
+  return <IntlProvider rtl>{props.children}</IntlProvider>;
+}
 ```
 
-## 3.postcss-rtl
+### 3.Post-Processing CSS
```
Comment (MohammadYounes) [inline src/pages/guide/rtl/en/index.md]: @simonguo We already switched to generating full RTL version, The use of bidi version will have some discrepancies and wont work well out of the box. I recommend keeping the officially supported method only.

**Review by MohammadYounes — CHANGES_REQUESTED** [review]
@simonguo   I believe we should mention the following:

1. Including the RTL version when using the compiled React Suite CSS
 `~rsuite/dist/styles/rsuite-default.rtl.css` or `~rsuite/dist/styles/rsuite-dark.rtl.css`

2. Processing the final CSS via [rtlcss](https://rtlcss.com) in case of using the Less version `~rsuite/lib/styles/themes/default/index.less` or `~rsuite/lib/styles/themes/dark/index.less`

**Comment by simonguo** [comment]
@MohammadYounes 
We are refactoring the documentation with Next.js and migrating to the [rsuite](https://github.com/rsuite/rsuite/tree/docs/docs) library.

Then modify the instructions related to rtl.

**Pull request closed without merging**